lab_share_lib/rabbit/async_consumer.py

```diff
@@ -268,7 +268,7 @@ def on_message(self, channel, basic_deliver, properties, body):
         :param bytes body: The message body
         """
         LOGGER.info(f"Received message # {basic_deliver.delivery_tag}")
-        MESSAGE_LOGGER.info(f"Received message # {basic_deliver.delivery_tag} with body:  {body.decode()}")
+        MESSAGE_LOGGER.info(f"Received message # {basic_deliver.delivery_tag} with body:  {body}")
```

Comment:
Why does this not need to be decoded any more?

Reply:
This logger message using decode() didnt work with binary format. I'll double check to see the output in both encodings to check it still works with json.

Reply:
Ok, confirmed that without decode() it works with both formats